bundles/org.openhab.persistence.jdbc/src/main/resources/OH-INF/i18n/jdbc-mariadb.properties

```
@@ -0,0 +1,4 @@
+# add-on
+
+addon.jdbc-mariadb.name = JDBC Persistence (SQLite)
```

```suggestion
addon.jdbc-mariadb.name = JDBC Persistence (MariaDB)
```

Will this file actually be used? What about other than MariaDB?